src/lib/Language/Coq/Pretty.hs

```diff
@@ -230,296 +223,189 @@ renderArgsWithOptionalType
   -- Special notation for somehting that looks like an operator an
   -- is applied to two arguments
-  renderGallina' p (App (Qualid op) (PosArg l :| [PosArg r])) | qualidIsOp op =
-    case lookup op precTable of
-      Just (n, LeftAssociativity) ->
-        maybeParen (n < p)
-          $   group
-          $   renderGallina' n l
-          </> renderQOp op
-          <!> renderGallina' (n + 1) r
-      Just (n, RightAssociativity) ->
-        maybeParen (n < p)
-          $   group
-          $   renderGallina' (n + 1) l
-          </> renderQOp op
-          <!> renderGallina' n r
-      Just (n, NoAssociativity) ->
-        maybeParen (n < p)
-          $   group
-          $   renderGallina' (n + 1) l
-          </> renderQOp op
-          <!> renderGallina' (n + 1) r
-      Nothing ->
-        maybeParen (p > defaultOpPrec)
-          $   group
-          $   renderGallina' (defaultOpPrec + 1) l
-          </> renderQOp op
-          <!> renderGallina' (defaultOpPrec + 1) r
-
-  renderGallina' p (App f args) =
-    maybeParen (p > appPrec)
-      $ let -- If we're providing a named argument, it turns out we can't use a
-            -- notation, so we avoid doing that for operator names in that case.
-            renderedFunction
-              | Qualid qf <- f
-              , any
-                (\case
-                  NamedArg _ _ -> True
-                  _            -> False
-                )
-                args
-              = renderGallina' appPrec qf
-              | otherwise
-              = renderGallina' appPrec f
-        in  renderedFunction </> align (renderArgs' (appPrec + 1) H args)
-
-  renderGallina' p (ExplicitApp qid args) =
-    maybeParen (p > appPrec)
-      $  "@"
-      <> renderGallina qid
-      <> softlineIf args
-      <> renderArgs' (appPrec + 1) H args
-
-  renderGallina' p (InScope tm scope) =
-    maybeParen (p > scopePrec)
-      $  renderGallina' scopePrec tm
-      <> "%"
-      <> renderIdent scope
-
+  renderGallina' p (App (Qualid op) (PosArg l :| [PosArg r]))
+    | qualidIsOp op = case lookup op precTable of
+      Just (n, LeftAssociativity) -> maybeParen (n < p)
+        $ group
+        $ renderGallina' n l </> renderQOp op <!> renderGallina' (n + 1) r
+      Just (n, RightAssociativity) -> maybeParen (n < p)
+        $ group
+        $ renderGallina' (n + 1) l </> renderQOp op <!> renderGallina' n r
+      Just (n, NoAssociativity) -> maybeParen (n < p)
+        $ group
+        $ renderGallina' (n + 1) l </> renderQOp op
+        <!> renderGallina' (n + 1) r
+      Nothing -> maybeParen (p > defaultOpPrec)
+        $ group
+        $ renderGallina' (defaultOpPrec + 1) l </> renderQOp op
+        <!> renderGallina' (defaultOpPrec + 1) r
+  renderGallina' p (App f args) = maybeParen (p > appPrec)
+    $     -- If we're providing a named argument, it turns out we can't use a
+          -- notation, so we avoid doing that for operator names in that case.
+    let renderedFunction
```

Comment:
In order for Floskell to associate the comment with the binding and not the `let` expression, the comment would have to be a Haddock comment. In this particular case a `where` clause is probably more appropriate, though.

```haskell
  renderGallina' p (App f args) = maybeParen (p > appPrec)
    $ renderedFunction </> align (renderArgs' (appPrec + 1) H args)
   where
     -- If we're providing a named argument, it turns out we can't use a
     -- notation, so we avoid doing that for operator names in that case.
     renderedFunction :: Doc
     renderedFunction
       | Qualid qf <- f, any isNamedArg args = renderGallina' appPrec qf
       | otherwise = renderGallina' appPrec f

     isNamedArg :: Arg -> Bool
     isNamedArg (NamedArg _ _) = True
     isNamedArg _ = False
```